src/components/PurchaseOrder/util.js

```diff
@@ -70,3 +70,11 @@ export const getAddresses = (addresses) => {
     };
   });
 };
+
+export const getVendors = (vendors) => {
+  return vendors.filter(vendor => !vendor.isVendor && vendor.status === 'Active')
```

Comment:
Please check the condition of filtering. I think you need only vendors here, or I'm wrong? 
https://github.com/folio-org/ui-invoice/blob/master/src/invoices/InvoiceForm/InvoiceForm.js#L134

Reply:
I need to shows active vendor organizations, and NOT non-vendor organizations.